### ISSUE TITLE
pulmonary cavity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6881]
**Preferred term label:**
pulmonary cavity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Gas-containing space; manifests as lucency or low attenuation within nodule, mass, or consolidation

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary thoracic imaging finding HP:0031983 (Can be seen on CT or chest XR)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Infection

- Necrotizing pneumonia
- Abscess
- Septic emboli

Neoplasia with cavitation

- Lung cancer, typically squamous cell carcinoma
- Metastases: Squamous cell, adenocarcinoma, sarcoma

Vasculitis

- Granulomatosis with polyangiitis

Pulmonary infarction

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
How is this different from https://hpo.jax.org/app/browse/term/HP:0032445 Pulmonary cyst? @eswietlik

**Comment by pnrobinson** [comment]
Also is this different from cavitating pulmonary mass (new term)? @eswietlik

**Comment by pnrobinson** [comment]
@eswietlik  -- please see above questions!

**Comment by eswietlik** [comment]
> How is this different from https://hpo.jax.org/app/browse/term/HP:0032445 Pulmonary cyst? @eswietlik

Pulmonary cavity is surrounded by mass, nodule, or consolidation that is creating wall thickness >2-4 mm. Pulmonary cyst on the other hand is thin-walled in most cases contains air.

**Comment by eswietlik** [comment]
> Also is this different from cavitating pulmonary mass (new term)? @eswietlik

A pulmonary cavity may occur be in nodule, mass, consolidation; so we could actually have a term pulmonary cavity as a parent term and the following child terms: cavitating pulmonary nodule, cavitating pulmonary mass, cavitating pulmonary mass.

**Comment by pnrobinson** [comment]
added
